core-common/src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java

```diff
@@ -2,6 +2,7 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Function;
```

Comment:
Seems that there is a typo - probably usual java Function will be enough

Reply:
👍